### PR TITLE
Having your hair turned purple by Poly Omers now shows a message

### PR DESCRIPTION
**Pull request by carlarctg**
## Why It's Good For The Game
you may not notice it

## Changelog
:cl:
add: Having your hair turned purple by Polypyrylium Oligomers now shows a message
/:cl:

### CODE REVIEW

**Comment by trollbreeder** [comment]
the quality of life master delivers a quality PR once again, thumbs up

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 1]
```diff
@@ -1295,6 +1295,10 @@
 	if(method == TOUCH || method == VAPOR)
 		if(M && ishuman(M) && reac_volume >= 0.5)
 			var/mob/living/carbon/human/H = M
+			var/mob_hair = H.hair_color
+			var/is_purple = "92f"
+			if(mob_hair != is_purple)
+				to_chat(H, "<span class='polypurple'>Your hair feels.. purple?</span>")
 			H.hair_color = "92f"
 			H.facial_hair_color = "92f"
```
Comment (Cyberboss) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: Can you cleanup the three instances of `"92f"` into one var please

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by cacogen** [comment]
i dont understand

**Comment by ExcessiveUseOfCobblestone** [comment]
define it

**Comment by ExcessiveUseOfCobblestone** [comment]
this should apply to the beard too? also stale.

**Pull request closed without merging**